x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/list_detail_page/list_details.cy.ts

```diff
@@ -91,4 +102,35 @@ describe('Exception list management page', () => {
     visitWithoutDateRange(exceptionsListDetailsUrl(getExceptionList1().list_id));
     cy.get(EXCEPTIONS_LIST_MANAGEMENT_DESCRIPTION).should('have.text', 'Add a description');
   });
+
+  it('Should create a new list and link it to two rules', () => {
+    createSharedExceptionList(
+      { name: 'Newly created list', description: 'This is my list.' },
+      true
+    );
+
+    // After creation - directed to list detail page
+    cy.get(EXCEPTIONS_LIST_MANAGEMENT_NAME).should('have.text', EXCEPTION_LIST_NAME);
+
+    // Open Link rules flyout
+    cy.get(EXCEPTION_LIST_DETAILS_LINK_RULES_BTN).click();
+
+    // Link the first Rule
+    clickOnLinkRulesByRuleRowOrderInListDetail(0);
+    // Link the second Rule
+    clickOnLinkRulesByRuleRowOrderInListDetail(1);
+
+    cy.get(MANAGE_RULES_SAVE).first().click();
```

Comment:
NIT: Wrap this into a task method.

Reply:
@WafaaNasr still seeing the old method here.

Reply:
Should be there now [d49bd16](https://github.com/elastic/kibana/pull/152757/commits/d49bd165495a5444379954df79eb982b8d1ef4dc)